runhouse/resources/module.py

```diff
@@ -220,11 +199,49 @@ def env(self):
     def env(self, new_env: Optional[Union[str, Env]]):
         self._env = _get_env_from(new_env)
 
+    @property
+    def signature(self):
+        return self._signature or {
+            name: self.method_signature(method)
+            for (name, method) in inspect.getmembers(self.__class__)
+            if not name[0] == "_" and name not in LOCAL_METHODS
+        }
+
+    def method_signature(self, method, rich=False):
+        """Extracts the properties of a method that we want to preserve when sending the method over the wire."""
+        if not callable(method):
+            return {
+                "signature": None,
+                "property": True,
+                "async": False,
+                "gen": False,
+                "local": False,
+            }
+
+        signature = inspect.signature(method)
+        ppts = {
+            "signature": str(signature),
+            "property": not callable(method),
+            "async": inspect.iscoroutinefunction(method)
+            or inspect.isasyncgenfunction(method),
+            "gen": inspect.isgeneratorfunction(method)
+            or inspect.isasyncgenfunction(method),
+            "local": "local" in signature.parameters
```

Comment:
It means that the method has a kwarg `local=True` which makes it run locally rather than as an RPC by default (unless overridden with `local=False`). Not very clear, gonna change it to `default_local`.